test/e2e/app/definition.go

```diff
@@ -250,50 +311,45 @@ func TestnetFromManifest(manifest types.Manifest, infd types.InfrastructureData,
 		})
 	}
 
-	return types.Testnet{
-		Network:      manifest.Network,
-		Testnet:      adaptCometTestnet(cmtTestnet, cfg.OmniImgTag),
-		OmniEVMs:     omniEVMS,
-		AnvilChains:  anvils,
-		PublicChains: publics,
-	}, nil
+	return publics, nil
 }
 
 // internalNetwork returns a internal intra-network netconf.Network from the testnet and deployInfo.
-func internalNetwork(testnet types.Testnet, deployInfo map[types.EVMChain]netman.DeployInfo, evmPrefix string,
-) netconf.Network {
+func internalNetwork(testnet types.Testnet, deployInfo map[types.EVMChain]netman.DeployInfo, evmPrefix string) netconf.Network {
 	var chains []netconf.Chain
 
-	omniEVM := omniEVMByPrefix(testnet, evmPrefix)
-	chains = append(chains, netconf.Chain{
-		ID:                omniEVM.Chain.ID,
-		Name:              omniEVM.Chain.Name,
-		RPCURL:            omniEVM.InternalRPC,
-		AuthRPCURL:        omniEVM.InternalAuthRPC,
-		PortalAddress:     deployInfo[omniEVM.Chain].PortalAddress.Hex(),
-		DeployHeight:      deployInfo[omniEVM.Chain].DeployHeight,
-		BlockPeriod:       omniEVM.Chain.BlockPeriod,
-		FinalizationStrat: omniEVM.Chain.FinalizationStrat,
-		IsOmni:            true,
-	})
-
-	// Add all anvil chains
-	for _, anvil := range testnet.AnvilChains {
+	if !testnet.OnlyMonitor {
```

Comment:
add comments what this switch does, this isn't obvious why we exclude some thing when this is true/false.